src/index.tsx

```diff
@@ -15,7 +15,7 @@ export {
   SkinTones,
   Theme,
   Categories,
-  EmojiClickData,
+  type EmojiClickData,
```

Comment:
@ealush this won't work without adding the type anotation and moving it to diffrent export not sure if it actually affecting the package types but I think it not